docs/environments/catch.md

```diff
@@ -17,4 +17,44 @@ firstpage:
 | Reward Space | Box(-10.0, [ 1. inf], (2,), float32) |
 | Import | `momaland.envs.catch_v0` |
 
-A Parallel Environment where drone learn how to surround a moving target trying to escape.
+A `Parallel` environment where drones learn how to surround a moving target trying to escape.
+
+## Observation Space
+The observation space is a continuous boxes with the length `(num_drones + 1) * 3` where each 3 values represent the coordinates of the drones in this order:
+- the agent.
+- the target.
+- the other agents.
+The higher bound is the `size` of the environment. Lower bound is `-size`.
+
+Example:
+`[x_0, y_0, z_0, x_targ, y_targ, z_targ, x_1, y_1, z_1, ..., x_n, y_n, z_n]`
+
+## Action Space
+The action space is a 3D vector representing the movement of the agent.
+The higher bound is `1`, the lower bound is `-1`.
+
+## Reward Space
+The reward space is a 2D vector containing rewards for:
+- Minimizing distance towards the target
+- Maximizing average distance towards other agents (avoiding collision).
+The higher bound is `[1, inf]`, the lower bound is `[-10, -10]`.
+
+## Starting State
+Where `size = 3`, the initial starting positions of the agents are `[0, 0, 1], [1, 1, 1], [0, 1, 1], [2, 2, 1]` while the target position is `[1, 1, 2.5]`
+
+## Episode Termination
+The episode is terminated if one of the following conditions are met:
+- 2 agents collide.
+- An agent and the target collide.
+- An agent collides with the ground.
+
+## Arguments
+- render_mode (str, optional): The mode to display the rendering of the environment. Can be human or None.
+- size (int, optional): Size of the area sides
+- num_drones (int, optional): Amount of drones
+- init_flying_pos (nparray[float], optional): 2d array containing the coordinates of the agents is a (3)-shaped array containing the initial XYZ position of the drones.
+- init_target_location (nparray[float], optional): Array of the initial position of the moving target
+- target_speed (float, optional): Distance traveled by the target at each timestep
+
+## Credits
+The code was adapted from: [Felten's source](https://github.com/ffelten/CrazyRL).
```

Comment:
Looks so weird to see my name written like this 😆

Reply:
For DST it said `Yang's source`, I am just copying you lol